content/updates/2017-02-09-release-notes.md

```diff
@@ -0,0 +1,22 @@
++++
+date = "2017-02-09"
+title = "Platform Release Notes: February 9, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+### Added
+- The [cloud.gov status page](https://cloudgov.statuspage.io/), which uses another domain to publicly report the availability and performance of cloud.gov, now includes the availability of the new, FedRAMP Authorized AWS GovCloud environment. It also reports the availability of the legacy AWS East/West environment as some users’ systems have not yet been migrated.
+
+### Fixed
+- When a user reaches the end of their activity log, the log no longer displays a “Show more” button. Previously this button implied there might be more content when there was not.
+- Users with administrative permissions can now see all of the spaces and apps they were administrators for on the overview page. Previously, only the most recent requests were providing data to this view.
```

Comment:
I might re-phrase this to add some clarity and remove some technical terms. Maybe:

> - Users with many organizations can now see all of the spaces and apps for which they are a member. Previously,  on the overview page, some organizations would appear empty or with some spaces and apps missing.